CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 ## 3.79.0 (Unreleased)
+NOTES:
+* spanner: `num_nodes` will become a required field in the next major release and a value has to be set on `google_spanner_instance`. It will also conflict with the new field `processing_units`. ([#9716](https://github.com/hashicorp/terraform-provider-google/pull/9716))
+
+FEATURES:
+* **New Resource:** `google_dialogflow_cx_entity_type` ([#9717](https://github.com/hashicorp/terraform-provider-google/pull/9717))
+* **New Resource:** `google_dialogflow_cx_page` ([#9683](https://github.com/hashicorp/terraform-provider-google/pull/9683))
+
+IMPROVEMENTS:
+* all (DCL resources only): ignore diff after apply errors coming from the DCL ([#9701](https://github.com/hashicorp/terraform-provider-google/pull/9701))
```

Comment:
Should we include this? It's sort of internal-only. If we include it, it might be nice to reword to be [more friendly to end users](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md#not-user-centric) like: `made permadiffs no longer cause applying a plan to crash` or similar. Maybe it's a bugfix instead of an improvement?

Reply:
I'd lean towards removing it, as I don't really have a good way of describing the situation where this would happen, and we don't have a good way to describe which resources would be impacted

Reply:
removed